src/components/utils/test/date_time_test.cc

```diff
@@ -33,6 +33,19 @@
 #include "gtest/gtest.h"
 #include "utils/date_time.h"
 
+#ifndef __linux__
+void usleep(int waitTime) {
```

Comment:
@OHerasym why don't you just use common `usleep`? BTW our `Thread` implementation provides it